src/plugins/graph/models/graph-model.ts

```diff
@@ -306,6 +327,20 @@ export const GraphModel = TileContentModel
     },
   }))
   .actions(self => ({
+    // Returns an objet's color, given its id.
+    // This is an action because if the id doesn't have a specified color, this will assign one to it.
+    getColorForId(id: string) {
+      let colorIndex = self._idColors.get(id);
+      if (colorIndex === undefined) {
+        withoutUndo();
```

Comment:
Can you add a comment here explaining what happens if withoutUndo isn't used?
I have a guess, but it'd be good to describe it, so if we have to change the withoutUndo behavior we know what problem it was solving in this case.